cdap-ui/app/features/tracker/controllers/entity-ctrl.js

```diff
@@ -15,14 +15,15 @@
  */
 
 class TrackerEntityController{
-  constructor($state, myJumpFactory, rDatasetType) {
+  constructor($state, myJumpFactory, rDatasetType, rSystemTags) {
     this.$state = $state;
     this.myJumpFactory = myJumpFactory;
 
     let entityParams = this.$state.params.entityType;
     let entitySplit = entityParams.split(':');
 
     this.datasetType = rDatasetType;
+    this.datasetExplorable = rSystemTags && rSystemTags.indexOf('explore') !== -1;
```

Comment:
It would be nicer if we could check `Array.isArray(rSystemTags)`. It communicates/verifies if its an array before we do `indexOf`